resources/cluster-users/values.yaml

```diff
@@ -57,6 +57,11 @@ clusterRoles:
       - "caching.internal.knative.dev"
       - "autoscaling.internal.knative.dev"
```

Comment:
Please update tests for that https://github.com/kyma-project/kyma/tree/master/resources/cluster-users/files